### ISSUE TITLE
[Feature request] Settings option to set visibility of replies to "not listed" automatically

### ISSUE THREAD

**Issue by rakoenig** [mastodon/mastodon#21084]
### Pitch

When replying to a public post, then the reply is also public. 

### Motivation

Some people would prefer to set replies to "not listed" by default, but there is no option available in the settings.

**Comment by hackbod** [comment]
I would add, based on comments from people saying unlisting comments is the normal recommended thing to do, it might make sense to have this be the default setting.

**Comment by trwnh** [comment]
Duplicate of #19685 

However, I'd also like to say that the recommendation for "unlisted comments" makes no sense -- public vs. unlisted only matters for top-level posts and self-replies. Replying as public doesn't actually do anything differently, as far as Mastodon's public timelines are concerned.

**Comment by velw** [comment]
> Duplicate of #19685
> 
> However, I'd also like to say that the recommendation for "unlisted comments" makes no sense -- public vs. unlisted only matters for top-level posts and self-replies. Replying as public doesn't actually do anything differently, as far as Mastodon's public timelines are concerned.

I'm not sure it's a duplicate. The issue you linked is talking about adding an additional option, so users have two settings for their default post privacy level.

If I've understood correctly, this request is about setting replies to "unlisted" by default. It sounds as though that would actually keep the post visibility setting more closely aligned with how replies are handled, as well, in terms of their visibility in local and federated timelines - which may also clear up some confusion among users as to the need to adjust this setting for replies in the first place.

When I first started using Mastodon a month or so ago, I saw a flurry of complaints from long-term users that new people weren't setting their replies to "unlisted" the way they 'should' and that this was interfering with people's timelines. So either someone is seeing them when they shouldn't, or someone is confused when they shouldn't be, and people are being told to do this by hand for every reply when it could just be the automatic, default behaviour. It seems like whatever is happening right now could be improved.
